prog/weaver/http.go

```diff
@@ -203,6 +205,18 @@ var statusTemplate = defTemplate("status", `\
             TTL: {{.DNS.TTL}}
         Entries: {{countDNSEntries .DNS.Entries}}
 {{end}}\
+{{if .Proxy}}\
+
+        Service: proxy
+        Address: {{printList .Proxy.Addresses}}
+{{end}}\
+{{if .Plugin}}\
+
+        Service: plugin (v{{.Plugin.Version}})
+{{if eq .Plugin.Version 1}}\
```

Comment:
Minor/Nitpick: replace `1` with a constant, esp. as used in `plugin/status.go` in `Version:        1,`